app/templates/layout.html

```diff
@@ -46,7 +48,12 @@
                         <li class="nav-item"><a class="nav-link" href="/register">Register</a></li>
                         <li class="nav-item"><a class="nav-link" href="/login">Log In</a></li>
                     </ul>
-                {% endif %}
+                {% endif %} -->
```

Comment:
Same as above, do not comment out unnecessary stuff, just delete. You can always retrieve this back when viewing this pull request.